src/devices/neo.ts

```diff
@@ -457,6 +457,74 @@ const definitions: DefinitionWithExtend[] = [
         description: 'Temperature & humidity sensor',
         extend: [temperature(), humidity(), battery()],
     },
+    {
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_1youk3hj']),
+        model: 'NAS-PS10B2',
+        vendor: 'NEO',
+        description: 'Human presence sensor',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            e.enum('presence_state', ea.STATE, ['none', 'presence']).withDescription('Presence State'),
+            e.enum('human_motion_state', ea.STATE, ['none', 'small', 'large']).withDescription('Human Motion State'),
+            e
+                .numeric('dis_current', ea.STATE)
+                .withUnit('cm')
+                .withValueMin(0)
+                .withValueMax(600)
+                .withValueStep(1)
+                .withLabel('Current distance')
+                .withDescription('Current Distance of Detected Motion'),
```

Comment:
```suggestion
                .withDescription('Current distance of detected motion'),
```

Could you fix the capitalisation? (also for the others)